### DISCUSSION TITLE
Where do I find the branch with __cacert_entrypoint.sh ?

### DISCUSSION THREAD

**Discussion by gumbo2k** [structurizr/lite#157] [General]
I found `__cacert_entrypoint.sh` in the current image of structurizr/lite. 
```
REPOSITORY                       TAG                      IMAGE ID       CREATED              SIZE
...
structurizr/lite                 2024.07.02               60385bed441b   7 weeks ago      489MB
structurizr/lite                 latest                   60385bed441b   7 weeks ago          489MB
```

And I found that this nice `__cacert_entrypoint.sh` script solves my issues with an intrusive TLS proxy that made downloads of themes impossible.

Now, I'd love to extend this script, but I can't find the upstream. The structurizr/lite repo seems to have just the `main` branch and that branch apparently doesn't have that script.

Can anybody point me to the upstream please?

**Comment by simonbrowndotje** [comment]
It's not something I've added ... I would take a look in the base image -> https://github.com/structurizr/lite/blob/main/Dockerfile#L1

**Reply by gumbo2k** [reply]
Yeap. 
That script is indeed [part of the eclipse-temurin base image](https://github.com/adoptium/containers/blob/057e5aa7581e96b8a2334290e750b329d62abfdf/21/jre/ubuntu/jammy/entrypoint.sh) . I should have figured that out by myself :laughing: . 
Sorry for bothering you. 

I'll check with adopium upstream if they are interested in adding a `RUN_AS=<uid>:<gid>`-feature to their container images.
Something to change the uid:gid before running the jvm, so that files created by the java process belong to that uid:gid.

That could replace the workaround I suggested in https://github.com/structurizr/structurizr.github.io/pull/20
